translations/messages+intl-icu.ru.xlf

```diff
@@ -306,7 +306,7 @@
             </trans-unit>
             <trans-unit id="post.num_comments">
                 <source>post.num_comments</source>
-                <target>{count, plural, one {# комментарий} few {# комментария} many {# комментариев}}</target>
+                <target>{count, plural, one {# комментарий} other {# комментариев}}</target>
```

Comment:
this change is wrong. Russian has 3 differents forms. `few` should not have been removed

Reply:
@stof The previous format doesn't work, what is the correct solution that you suggest?

Reply:
well, `many` should have been `other`. But the `few` variant is still necessary for proper Russian pluralization AFAIK.

Reply:
Got it! fixed in https://github.com/symfony/demo/pull/934